crates/swc_ecma_minifier/src/compress/optimize/inline.rs

```diff
@@ -323,7 +322,7 @@ impl Optimizer<'_> {
                 && usage.declared
                 && may_remove
                 && !usage.reassigned
-                && (usage.can_inline_var() || usage.is_mutated_only_by_one_call())
+                && usage.assign_count == 0
```

Comment:
There's no need to check if a used only once var is mutated -- we inline it anyway.